<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>545</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="MyTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>251</width>
      <height>571</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Входной</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Выходной</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="spinBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>50</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Кол-во элементов&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="FillButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>257</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сгенерировать массив</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>257</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Простой выбор</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пузырьковая сортировка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Простая вставка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Пирамидальная сортировка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сортировка методом Шелла</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Быстрая сортировка</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Сортировка слиянием</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Задание</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="SortButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>257</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Сортировать</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ClearButton">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>190</y>
      <width>257</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
